doc/guide/python_quickstart.hpp

```diff
@@ -119,17 +119,17 @@ list of all the mlpack functionality offered through Python, split into some
+   href="http://www.mlpack.org/doc/mlpack-git/python/adaboost.html">adaboost()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/decision_stump.html">decision_stump()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/decision_tree.html">decision_tree()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/hmm_train.html">hmm_train()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/hmm_generate.html">hmm_generate()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/hmm_loglik.html">hmm_loglik()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/hmm_viterbi.html">hmm_viterbi()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/hoeffding_tree.html">hoeffding_tree()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/logistic_regression.html">logistic_regression()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/nbc.html">nbc()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/perceptron.html">perceptron()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/random_forest.html">random_forest()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/softmax_regression.html">softmax_regression()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/cf.html">cf()</a></tt>
 
- - Distance-based problems: <tt><a href="http://www.mlpack.org/docs/mlpack-git/python/approx_kfn.html">approx_kfn()</a></tt>, <tt><a href="http://www.mlpack.org/docs/mlpack-git/python/emst.html">emst()</a></tt>, <tt><a href="http://www.mlpack.org/docs/mlpack-git/python/fastmks.html">fastmks()</a></tt>, <tt><a href="http://www.mlpack.org/docs/mlpack-git/python/kfn.html">kfn()</a></tt>, <tt><a href="http://www.mlpack.org/docs/mlpack-git/python/knn.html">knn()</a></tt>, <tt><a href="http://www.mlpack.org/docs/mlpack-git/python/krann.html">krann()</a></tt>, <tt><a href="http://www.mlpack.org/docs/mlpack-git/python/lsh.html">lsh()</a></tt>, <tt><a href="http://www.mlpack.org/docs/mlpack-git/python/det.html">det()</a></tt>
+ - Distance-based problems: <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/approx_kfn.html">approx_kfn()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/emst.html">emst()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/fastmks.html">fastmks()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/kfn.html">kfn()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/knn.html">knn()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/krann.html">krann()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/lsh.html">lsh()</a></tt>, <tt><a href="http://www.mlpack.org/doc/mlpack-git/python/det.html">det()</a></tt>
```

Comment:
Same for this file.

Reply:
Yeah, I updated this to point to the Python documentation page (and same for the CLI documentation) instead of listing each binding.  It should be easier to maintain now too.